src/index.ts

```diff
@@ -78,4 +83,9 @@ const specLikeCodec = union([
 	type({
 		asyncapi: literal('2.0.0'),
 	}),
+	type({
+		meta: type({
```

Comment:
I don't understand the naming. Why this is called `meta`? Shouldn't it be 'sketch'?

Reply:
It's part of sketch file format spec

Reply:
I mean, how do we know later, that this one is related to sketch? Maybe leave a comment or smth?

Reply:
Ok, I'll a comment. Please proceed with the rest of the review, don't stop on a single comment.

Reply:
adde a comment